kytos/core/controller.py

```diff
@@ -128,6 +129,26 @@ def __init__(self, options=None, loop=None):
         #: Now you can access the enabled napps with:
         #: from napps.<username>.<napp_name> import ?....
         sys.path.append(os.path.join(self.options.napps, os.pardir))
+        sys.excepthook = self.exhandler
+
+    # pylint: disable=invalid-name,no-self-use
+    def exhandler(self, exctype, value, tb):
+        """Define exception hook handler
```

Comment:
The current docstring is saying just a bit over what can be inferred from the function definition line: `def exhandler -> Define exception hook handler`. It'd be better if it said *what* this handler does, e.g.: "Print uncaught exceptions and/or save them to a log file."